draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -120,35 +120,35 @@
-        ASPA-based AS_PATH verification provides detection and mitigation of route leaks and improbable AS paths.
-        It also provides protection, to some degree, against prefix hijacks with forged-origin or forged-path-segment (<xref target="use-cases"/>).
-        These new ASPA-based procedures automatically detect such anomalous AS_PATHs in announcements that are advertised between AS.
+        ASPA-based AS_PATH verification provides detection and mitigation of route leaks.
+        It also provides protection, to some degree, against prefix hijacks with forged-origin or forged-path-segment (<xref target="property"/>).
+        These new ASPA-based procedures automatically detect such anomalous AS_PATHs in eBGP announcements.
       </t>
       <t>
         ASPA objects are cryptographically signed registrations of customer-to-provider relationships and stored in a distributed database <xref target="I-D.ietf-sidrops-aspa-profile"/>.
-        ASPA-based path verification is an incrementally deployable technique and provides benefits to early adopters in the context of limited deployment.
+        ASPA-based path verification is an incrementally deployable technique and provides benefits to early adopters in the context of limited deployment (<xref target="property"/>).
       </t>
       <t>
-        The verification procedures described in this document MUST be applied to BGP routes only with {AFI, SAFI} combinations {AFI 1 (IPv4), SAFI 1} and {AFI 2 (IPv6), SAFI 1} [IANA-AF].
+        The verification procedures described in this document MUST be applied to BGP routes with {AFI, SAFI} combinations {AFI 1 (IPv4), SAFI 1} and {AFI 2 (IPv6), SAFI 1} <xref target="IANA-AF"/> <xref target="IANA-SAF"/>.
         The procedures MUST NOT be applied to other address families by default.
       </t>
-
     <section title="Anomaly Propagation" anchor="propagation">
       <t>
-        Both route leaks and hijacks have similar effects on ISP operations - they redirect traffic and can result in denial of service (DoS), eavesdropping, increased latency, and packet loss.
+        Both route leaks and hijacks have similar effects on ISP operations.
+        They redirect traffic and can result in denial of service (DoS), eavesdropping, increased latency, and packet loss.
         The level of risk, however, depends significantly on the extent of propagation of the anomalies.
-        For example, a route leak or hijack that is propagated only to customers may cause bottlenecking within a particular ISP's customer cone, but if the anomaly propagates through lateral (i.e., non-transit) peers and transit providers, or reaches global distribution through transit-free networks, then the ill effects will likely be amplified and experienced across continents.
```

Comment:
top-tier - good, but second part is not. It is DFZ, default free networks, not transit-free connections/networks

Reply:
Accepted. Claudio also had comments to make it better. My local updated version now reads:
"For example, a route leak that is propagated only to
   customers may cause bottlenecking within a particular ISP's customer
   cone, but if the anomaly propagates through lateral (i.e., non-
   transit) peers and transit providers at the top-tier, then the ill
   effects will likely be amplified and experienced worldwide.'